build.gradle

```diff
@@ -19,15 +19,17 @@ configurations {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
+    implementation 'junit:junit:4.13.1'
+    implementation 'junit:junit:4.13.1'
     implementation 'org.twitter4j:twitter4j-core:4.0.7'
     implementation 'org.apache.commons:commons-lang3:3.12.0'
     implementation 'io.dropwizard:dropwizard-core:2.0.29'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
```

Comment:
org.junit.jupiter:junit-jupiter-api and org.junit.jupiter:junit-jupiter-engine added twice ?